src/main/java/org/folio/dao/transactions/BaseTransactionDAO.java

```diff
@@ -56,22 +62,27 @@ public Future<List<Transaction>> getTransactions(List<String> ids, DBClient clie
   private Handler<AsyncResult<Results<Transaction>>> handleGet(Promise<List<Transaction>> promise) {
     return reply -> {
       if (reply.failed()) {
+        logger.error("Getting transactions failed", reply.cause());
```

Comment:
can add ids to this log as well

Reply:
We may not have Ids here. And we log with debug level the query criterion on previous step.